integration/vite-build-test.ts

```diff
@@ -76,6 +109,25 @@ test.describe("Vite build", () => {
     appFixture.close();
   });
 
+  test("server code is removed from client assets", async () => {
+    let publicBuildDir = path.join(fixture.projectDir, "public/build");
+
+    // detect client asset files
+    let assetFiles = glob.sync("**/*.@(js|jsx|ts|tsx)", {
+      cwd: publicBuildDir,
+      absolute: true,
+    });
+
+    // grep for server-only values in client assets
+    let result = shell
+      .grep("-l", /SERVER_ONLY_1|SERVER_ONLY_2/, assetFiles)
```

Comment:
Nice!

Reply:
Should this be simplified to target just `"SERVER_ONLY"` for both exports rather than numbering them? Makes modifying this test less error prone.